app/validators/validate-response-format.ts

```diff
@@ -1,32 +1,15 @@
 import { isEmpty } from '@ember/utils';
-import ContributorModel from 'ember-osf-web/models/contributor';
+import { FileReference } from 'ember-osf-web/packages/registration-schema';
 
-const isContributorModel = (input: any) => {
-    if (input instanceof ContributorModel) {
-        return true;
-    }
-    return false;
-};
-
-export function validateResponseFormat(opts: {format: string}) {
-    return (value: any) => {
-        if (opts.format) {
-            switch (opts.format) {
-            case 'file':
-                if (!isEmpty(value.id) && !isEmpty(value.name)) {
-                    return true;
+export function validateFileReference() {
+    return (_: string, newValue: FileReference[]) => {
+        if (newValue) {
+            for (const file of newValue) {
+                if (isEmpty(file.file_id) && isEmpty(file.file_name)) {
```

Comment:
Need to also invert the binary operator:
```suggestion
                if (isEmpty(file.file_id) || isEmpty(file.file_name)) {
```